app/scripts/templates/settings/account_recovery/recovery_key.mustache

```diff
@@ -0,0 +1,33 @@
+<div id="account-recovery-key">
+    <section class="modal-panel">
+        <div class="error"></div>
+
+        <h2>{{#t}}Recovery key{{/t}}</h2>
+
+        <p class="description">
+            <span class="note">{{#t}}NOTE:{{/t}}</span>
+            {{#t}}Keep your recovery key in a safe place where you can access it later. You will not be able to access this page again. You can download or print your key or email it to someone you trust.{{/t}}
+        </p>
+
+        <div class="recovery-key-container">
+            <div class="recovery-key">{{recoveryKey}}</div>
+        </div>
+
+        <div class="options">
+            <div class="title-container">
+                <div class="line">&boxh;&boxh;&boxh;&nbsp;</div>
+                <div class="title">{{#t}}Download your recovery key or print it{{/t}}</div>
+                <div class="line">&nbsp;&boxh;&boxh;&boxh;</div>
+            </div>
+
+            <div class="buttons">
+                <button class="download-key"><img class="recovery-icon-option" src="/images/recovery_key_download.svg">{{#t}}Download{{/t}}</button>
+                <button class="print-key"><img class="recovery-icon-option" src="/images/recovery_key_print.svg">{{#t}}Print{{/t}}</button>
+            </div>
+        </div>
+
+        <div class="button-row">
+            <button class="secondary-button done-link">{{#t}}Done, Recovery Key Set{{/t}}</button>
```

Comment:
What might be a nice touch is to convert this from a secondary button to a primary button once the user either downloads or prints. The secondary button seems a little strange to me though.

Reply:
That might be the intended flow, I'll follow up in https://github.com/mozilla/fxa-content-server/issues/6394.